apps/core/Results.js

```diff
@@ -0,0 +1,121 @@
+import type { ResultsQueryResponse } from './__generated__/ResultsQuery.graphql';
+
+type Props = {|
+  +travelFrom: string,
+  +travelTo: string,
+  +dateFrom: string,
+  +dateTo: string,
+|};
+
+type ItemProps = {|
+  node: ?{|
+    +id: string,
+    +price: ?number,
+    +flyFrom: ?string,
+    +flyTo: ?string,
+    +localDeparture: ?string,
+    +localArrival: ?string,
+  |},
+|};
+
+type ResultItemProps = {|
+  item: ItemProps,
+|};
+
+export default class Results extends React.Component<Props> {
+  resultItem = ({ item }: ResultItemProps) => {
+    if (!item.node) {
+      return null;
+    }
+    const { flyFrom, flyTo, localDeparture, localArrival, price } = item.node;
+    // TODO: properly handle possible undefined props in final version of list (for example: should be entry with undefined price still displayed?)
+    return (
+      <Text style={styles.listItem}>{`${flyFrom ||
+        'Unknown destination'} / ${localDeparture ||
+        'Unknown departure'} -> ${flyTo ||
+        'Unknown destination'} / ${localArrival ||
+        'Unknown arrival'}  | Price: ${price || 'Unknown price'}`}</Text>
+    );
+  };
+
+  keyExtractor = (item: ItemProps, index: number) => item.node?.id;
+
+  renderInner = (props: ResultsQueryResponse) => {
+    const { searchItineraries } = props;
+    if (
+      !searchItineraries ||
+      !searchItineraries.edges ||
+      searchItineraries.edges.length === 0
+    ) {
+      return <EmptyResults />;
+    }
+    return (
+      <>
+        <Text>{`Total results: ${searchItineraries.edges.length}`}</Text>
+        <FlatList
+          data={searchItineraries.edges}
+          keyExtractor={this.keyExtractor}
+          renderItem={this.resultItem}
+        />
+      </>
+    );
+  };
+
+  render() {
+    const { travelFrom, travelTo, dateFrom, dateTo } = this.props;
+    return (
+      <View style={styles.container}>
+        <Text>{`Search params: ${travelFrom} -> ${
+          travelTo ? travelTo : 'anywhere'
+        } | ${dateFrom} -> ${dateTo ? dateTo : 'anytime'}`}</Text>
+        <QueryRenderer
+          query={graphql`
+            query ResultsQuery($input: ItinerariesSearchInput!) {
+              searchItineraries(input: $input) {
+                edges {
+                  node {
+                    id
+                    price
+                    flyFrom
+                    flyTo
+                    localDeparture
+                    localArrival
+                  }
+                }
+              }
+            }
+          `}
+          variables={{
+            input: {
+              travelFrom,
+              travelTo,
+              dateFrom,
+              dateTo,
+            },
+          }}
+          render={this.renderInner}
+        />
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  listItem: {
+    fontSize: 12,
+    padding: 2,
+    margin: 2,
+    backgroundColor: '#dddddd',
```

Comment:
Also, this should probably be a value from orbit-tokens, possibly `$backgroundBody`

Reply:
Ok, swapped for `orbit-tokens` value for now.